ResearchKit/Common/ORKAnswerFormat.h

```diff
@@ -84,6 +85,17 @@ typedef NS_ENUM(NSInteger, ORKChoiceAnswerStyle) {
     ORKChoiceAnswerStyleMultipleChoice
 } ORK_ENUM_AVAILABLE;
 
+/// An enumeration of the format styles available for scale answers.
+typedef NS_ENUM(NSInteger, ORKNumberFormattingStyle) {
+    /// The default decimal style.
+    ORKNumberFormattingStyleDefault,
+    
+    /// Percent style.
+    ORKNumberFormattingStylePercent,
+    
+    /// Currency style.
+    ORKNumberFormattingStyleCurrency
```

Comment:
How about I add another property for overriding the locale used by the number formatter, along with a corresponding property on ORKScaleQuestionResult?

NSNumberFormatter's currencyCode by itself wouldn't account for other formatting issues like grouping/decimal separators, whether to show fractional units, etc. and this would allow folks to override the locale for decimal numbers too, if necessary.